testing/dub/APKBUILD

```diff
@@ -0,0 +1,26 @@
+# Contributor: Mathias LANG <pro.mathias.lang@gmail.com>
+# Maintainer: Mathias LANG <pro.mathias.lang@gmail.com>
+pkgname=dub
+pkgver=1.18.0
+pkgrel=0
+pkgdesc="Package and build management system for D"
+url="http://code.dlang.org/"
+arch="x86_64"
+license="MIT"
+depends="libcurl"
+makedepends="ldc ldc-static bash curl-dev"
+source="$pkgname-$pkgver.zip::https://github.com/dlang/dub/archive/v$pkgver.zip"
```

Comment:
There is a .tar.gz tarball autogenerated by GitHub like the .zip, isn't it usable ?